include/envoy/runtime/runtime.h

```diff
@@ -208,10 +209,10 @@ class Snapshot {
   /**
    * Fetch raw runtime data based on key.
    * @param key supplies the key to fetch.
-   * @return const std::string& the value or empty string if the key does not exist.
+   * @return absl::nullopt if the key does not exist or reference to the value std::string.
    */
-  virtual const std::string& get(absl::string_view key,
-                                 const std::string& default_value) const PURE;
+  virtual absl::optional<std::reference_wrapper<const std::string>>
```

Comment:
nit: per convention elsewhere this would be `StringOptRef` or `ConstStringOptRef`

Reply:
Done